about/design-goals.md

```diff
@@ -0,0 +1,29 @@
+# CloudEvents Specification Design Goals
+
+CloudEvents are typically used in a distributed system to allow for services to be loosely coupled during development, then then configured with tight-coupling for specific applications.
+
+The goal of the CloudEvents specification is to allow for different cloud providers to create interoperable event systems, that allow for third-party tools and services to provide:
```

Comment:
This seems a bit too specific to cloud providers. Perhaps change the emphasis onto event systems rather than cloud providers?
```The goal of the CloudEvents specification is to define interoperability of event systems that allow cloud providers, third-party tools and services to provide:```